CONTRIBUTING.md

```diff
@@ -72,6 +74,6 @@ Refer to [this document](https://docs.nearprotocol.com/docs/contribution/nearcor
 
 Once your change ends up in master, it will be released with the rest of the changes by other contributors on the regular release schedules.
 
-You should expect the changes from `master` to get merged into `beta` branch the next time `nightly` test run completes, assuming it passes. 
+You should expect the changes from `master` to get merged into `beta` branch the next time `nightly` test run completes, assuming it passes.
```

Comment:
Actually I realized that what is described here is completely outdated. I should update it in another PR